doc/api/fs.md

```diff
@@ -2066,16 +2066,22 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `mode` {integer} Not supported on Windows. **Default:** `0o777`.
+* `options` {Object|string}
+  * `parent` {boolean} **Default:** `false`.
```

Comment:
A tiny nit: we usually add periods after the ` **Default:** ...` parts only if they followed full sentences (the same for changes below).